pkg/oc/cli/debug/debug.go

```diff
@@ -828,3 +834,44 @@ func (o *DebugOptions) approximatePodTemplateForObject(object runtime.Object) (*
 
 	return nil, fmt.Errorf("unable to extract pod template from type %v", reflect.TypeOf(object))
 }
+
+// podContainerRunning returns false until the named container has ContainerStatus running (at least once),
+// and will return an error if the pod is deleted, runs to completion, or the container pod is not available.
+func podContainerRunning(containerName string) watch.ConditionFunc {
```

Comment:
are these the same?  oc/lib/conditions

Reply:
Thanks, moved this helper to the `oc/lib/conditions` package